<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KlientWindow</class>
 <widget class="QMainWindow" name="KlientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Konto klienta</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>511</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Wyszukiwanie zlecenia po numerze zlecenia</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>391</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Prosze podac identyfikator zlecenia</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_zlecenie">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>70</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Identyfikator zlecenia</string>
     </property>
    </widget>
    <widget class="QPushButton" name="szukaj_zlecenie">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>40</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Szukaj</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>511</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Wyszukiwanie zlecenia po identyfikatorze klienta</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>391</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Prosze podac identyfikator klienta</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_klient">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>70</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Identyfikator klienta</string>
     </property>
    </widget>
    <widget class="QPushButton" name="szukaj_klient">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>40</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Szukaj</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>731</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>350</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Efekty wyszukiwania</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zakoncz_button">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>570</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Zakoncz</string>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>label</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>tableView</zorder>
   <zorder>label_6</zorder>
   <zorder>zakoncz_button</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
